hummingbird/ml/operator_converters/svc.py

```diff
@@ -0,0 +1,117 @@
+# -------------------------------------------------------------------------
```

Comment:
Can we call the file `sv`? We will add regression sooner or later.

Reply:
done